GitUI/CommandsDialogs/RepoHosting/CreatePullRequestForm.cs

```diff
@@ -132,31 +132,37 @@ private void PopulateBranchesComboAndEnableCreateButton(IHostedRemote remote, Co
                     {
                         await TaskScheduler.Default;
 
-                        IHostedRepository hostedRepository = remote.GetHostedRepository();
-                        var branches = hostedRepository.GetBranches();
+                        try
+                        {
+                            IHostedRepository hostedRepository = remote.GetHostedRepository();
+                            var branches = hostedRepository.GetBranches();
 
-                        await this.SwitchToMainThreadAsync();
+                            await this.SwitchToMainThreadAsync();
 
-                        comboBox.Items.Clear();
+                            comboBox.Items.Clear();
 
-                        var selectItem = 0;
-                        var defaultBranch = hostedRepository.GetDefaultBranch();
-                        for (var i = 0; i < branches.Count; i++)
-                        {
-                            if (branches[i].Name == defaultBranch)
+                            var selectItem = 0;
+                            var defaultBranch = hostedRepository.GetDefaultBranch();
+                            for (var i = 0; i < branches.Count; i++)
                             {
-                                selectItem = i;
+                                if (branches[i].Name == defaultBranch)
+                                {
+                                    selectItem = i;
+                                }
+
+                                comboBox.Items.Add(branches[i].Name);
                             }
 
-                            comboBox.Items.Add(branches[i].Name);
-                        }
+                            if (branches.Count > 0)
+                            {
+                                comboBox.SelectedIndex = selectItem;
+                            }
 
-                        if (branches.Count > 0)
+                            _createBtn.Enabled = true;
+                        }
+                        catch (Exception)
                         {
-                            comboBox.SelectedIndex = selectItem;
                         }
```

Comment:
This isn't good. @mstv and I have been working to remove these silent catch blocks. 
The error must be bubbled up to the bug reporter, and explain the user what can be done to resolve it, if possible.